bolt/src/main/java/com/slack/api/bolt/middleware/builtin/SingleTeamAuthorization.java

```diff
@@ -22,6 +28,9 @@
     private final AppConfig appConfig;
     private final InstallationService installationService;
 
+    private Optional<AuthTestResponse> cachedAuthTestResponse = Optional.empty();
+    private AtomicLong lastCachedMillis = new AtomicLong(0L);
```

Comment:
Thanks to @aoberoi 's suggestion here: https://github.com/slackapi/java-slack-sdk/issues/468#issuecomment-650254481, we can remove this TTL and store the response when booting a Bolt app as with Bolt for JS.

Reply:
I came to think removing TTL from only single team auth may be confusing. I decided to add a "permanent" option by giving a negative TTL value instead.